docs/documentation/securing_apps/topics/oidc/javascript-adapter.adoc

```diff
@@ -230,34 +179,28 @@ While this mode is easy to set up, it also has some disadvantages:
 
-The alternative mode `cordova-native` takes a different approach.
-It opens the login page using the system's browser.
-After the user has authenticated, the browser redirects back into the app using a special URL.
-From there, the {project_name} adapter can finish the login by reading the code or token from the URL.
+The alternative mode `cordova-native` takes a different approach as it opens the login page using the system's browser. After the user has authenticated, the browser redirects back into the app using a special URL. From there, the {project_name} adapter can finish the login by reading the code or token from the URL.
 
-You can activate the native mode by passing the adapter type `cordova-native` to the `init` method:
+You can activate the native mode by passing the adapter type `cordova-native` to the `init()` method:
 
 [source,javascript]
 ----
 keycloak.init({
     adapter: 'cordova-native'
-})
+});
 ----
 
-This adapter required two additional plugins:
+This adapter requires two additional plugins:
 
 * https://github.com/google/cordova-plugin-browsertab[cordova-plugin-browsertab]: allows the app to open webpages in the system's browser
 * https://github.com/e-imaxina/cordova-plugin-deeplinks[cordova-plugin-deeplinks]: allow the browser to redirect back to your app by special URLs
 
 The technical details for linking to an app differ on each platform and special setup is needed.
 Please refer to the Android and iOS sections of the https://github.com/e-imaxina/cordova-plugin-deeplinks/blob/master/README.md[deeplinks plugin documentation] for further instructions.
 
-There are different kinds of links for opening apps: custom schemes (i.e. `myapp://login` or `android-app://com.example.myapp/https/example.com/login`) and https://developer.apple.com/ios/universal-links/[Universal Links (iOS)]) / https://developer.android.com/training/app-links/deep-linking[Deep Links (Android)].
-While the former are easier to set up and tend to work more reliably, the later offer extra security as they are unique and only the owner of a domain can register them.
-Custom-URLs are deprecated on iOS.
-We recommend that you use universal links, combined with a fallback site with a custom-url link on it for best reliability.
+There are different kinds of links for opening apps: custom schemes (i.e. `myapp://login` or `android-app://com.example.myapp/https/example.com/login`) and https://developer.apple.com/ios/universal-links/[Universal Links (iOS)]) / https://developer.android.com/training/app-links/deep-linking[Deep Links (Android)]. While the former are easier to set up and tend to work more reliably, the later offer extra security as they are unique and only the owner of a domain can register them. Custom-URLs are deprecated on iOS. We recommend that you use universal links, combined with a fallback site with a custom-url link on it for best reliability.
```

Comment:
```suggestion
Different kinds of links exist for opening apps: 
* custom schemes, such as `myapp://login` or `android-app://com.example.myapp/https/example.com/login`
* https://developer.apple.com/ios/universal-links/[Universal Links (iOS)]) / https://developer.android.com/training/app-links/deep-linking[Deep Links (Android)].
While the former are easier to set up and tend to work more reliably, the latter offer extra security because they are unique and only the owner of a domain can register them. Custom-URLs are deprecated on iOS.  For best reliability, we recommend that you use universal links combined with a fallback site that uses a custom-url link.
```